editor/src/clj/editor/code/preprocessors.clj

```diff
@@ -144,20 +143,18 @@ return m
                                            (map class)
                                            (g/node-value code-preprocessors :lua-preprocessors))]
     (when (not= old-lua-preprocessor-classes new-lua-preprocessor-classes)
-      (let [create-infos (try-create-lua-preprocessors lua-preprocessor-classes)
-            errors (filter #(= :error (:type %)) create-infos)]
-        (if (seq errors)
-          (report-error! "Failed to construct Lua preprocessors" (map :class-name errors))
-          (let [new-lua-preprocessors (success-values create-infos)]
-            (g/set-property! code-preprocessors :lua-preprocessors new-lua-preprocessors)))))))
+      (let [{:keys [created-lua-preprocessors faulty-class-names]}
+            (try-create-lua-preprocessors lua-preprocessor-classes)]
+        (if (seq faulty-class-names)
```

Comment:
Nitpick: I think you can use `faulty-class-names` instead of `(seq faulty-class-names)` here — because of group-into, this key will be either absent or contain a non-empty vector.